### PR TITLE
Task/14282 UI tests

### PR DESCRIPTION
**Pull request by flxschmidt**
## Description
UI and Screenshot Tests

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14282

### CODE REVIEW

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/TestPlans/Screenshots.xctestplan [hunk 1]
```diff
@@ -61,6 +61,7 @@
         "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_negative()",
         "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_pending()",
         "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_positive()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshpt_srs_positiveWithoutResultInTheApp()",
```
Comment (puneetmahali1) [inline src/xcode/ENA/TestPlans/Screenshots.xctestplan:65]:
```suggestion
        "ENAUITests_04a_ExposureSubmission\/test_screenshot_srs_positiveWithoutResultInTheApp()",
```

File: src/xcode/ENA/TestPlans/UITests.xctestplan [hunk 2]
```diff
@@ -111,6 +111,7 @@
         "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_available()",
         "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_pending()",
         "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_positive()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshpt_srs_positiveWithoutResultInTheApp()",
```
Comment (puneetmahali1) [inline src/xcode/ENA/TestPlans/UITests.xctestplan:115]:
```suggestion
        "ENAUITests_04a_ExposureSubmission\/test_screenshot_srs_positiveWithoutResultInTheApp()",
```

**Review by puneetmahali1 — APPROVED** [review]
LGTM! Just a minor typo.

File: src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift [hunk 3]
```diff
@@ -249,8 +249,68 @@ class ENAUITests_04a_ExposureSubmission: CWATestCase {
...
+		
+		// SRS Submission done and show Thank You Screen
+		XCTAssertTrue(
+			app.staticTexts[AccessibilityIdentifiers.ExposureSubmissionSuccess.description]
+				.waitForExistence(timeout: .medium)
+		)
+	}
+
+	/*
+	// Commented out in favour of SRS story
```
Comment (naveeddotio) [inline src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift:313]: Perhaps we need another task to fix the existing UI tests which we commented out. We need to find another way to mock the tests as we have removed TAN.

Comment (flxschmidt) [reply src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift:313]: Yes, I would also suggest to move that part into another task.

File: src/xcode/ENA/TestPlans/Screenshots.xctestplan (hunk 1 above)
Comment (flxschmidt) [reply src/xcode/ENA/TestPlans/Screenshots.xctestplan:65]: Eagle Eye :)

File: src/xcode/ENA/TestPlans/UITests.xctestplan (hunk 2 above)
Comment (flxschmidt) [reply src/xcode/ENA/TestPlans/UITests.xctestplan:115]: 👍

**Pull request merged**